### PR TITLE
inital readme

### PR DESCRIPTION
**Pull request by LePtitJoNadeau**
Add readme file. 

Of course, there is other things that can be add. 

You are welcome to add anything that you think could help!

### CODE REVIEW

**Comment by LePtitJoNadeau** [comment]
Well, I was waiting for Swyter review before merging. Is that okay with to marge this PR @sphinx856 ?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+## Prerequisites and Installing
+
+1. Sphinx and the Cursed Mummy: [Authoring Tools](https://store.steampowered.com/app/750430/Sphinx_and_the_Cursed_Mummy_Authoring_Tools/)
+
+For more information look at the [quick start guide](https://steamcommunity.com/sharedfiles/filedetails/?id=1306820082)
+
+[comment]: <> (TODO: ## Running the tests)
+
+[comment]: <> (TODO: ## Deployment)
```
Comment (Swyter) [inline README.md]: The mod can probably get deployed to Steam using SteamCMD in Travis CI, if we want to get fancy. I already have that working in a different Mount&Blade mod. But this is a very long-term plan.

**Comment by Swyter** [comment]
Looks good to me. I'm not sure the mod would need any tests, but it's fine.

**Comment by LePtitJoNadeau** [comment]
I remove the test section!

**Comment by Swyter** [comment]
@LePtitJoNadeau Great, thanks!

**Pull request merged**